pkg/cri/resource-manager/policy/builtin/memtier/pools_test.go

```diff
@@ -33,23 +33,37 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
 )
 
-func findNodeWithID(id int, nodes []Node) *Node {
+func findNodeWithID(id int, nodes []Node) Node {
 	for _, node := range nodes {
 		if node.NodeID() == id {
-			return &node
+			return node
 		}
 	}
 	panic("No node found")
 }
 
 func setLinks(nodes []Node, tree map[int][]int) {
+	parents := map[int]int{}
```

Comment:
Same comment than in #372: how about `nonOrphans := make(map[int]struct{})` or something similar(?)

Reply:
I'll think about something. nonOrphans is not a good choice though, IMO. We do collect parents in that map, basically constructing an alternative representation of the input `tree`, using per-node parent links instead of per-node children. Parent id being collected in vain as unused is a valid comment, though.